rebar.config

```
@@ -21,6 +21,8 @@
 
 {deps, [{jsx, "3.1.0"}, {cowboy, "2.10.0"}, {ranch, "2.1.0"}, {trails, "2.3.1"}]}.
 
+{pre_hooks, [{compile, "priv/update_swagger_url"}]}.
```

```suggestion
{pre_hooks, [{compile, "priv/update-swagger-url"}]}.
```
I'm not entirely sure why, but I prefer kebab-case for script names. Not a big deal, tho.

---

Done!